styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -184,7 +188,7 @@ protected void init(Environment environment) {
     when(requestAuthenticator.authenticate(any())).thenReturn(() -> Optional.of(idToken));
     WorkflowResource workflowResource =
         new WorkflowResource(storage, workflowValidator, workflowInitializer, workflowConsumer,
-            workflowActionAuthorizer);
+            workflowActionAuthorizer, clock);
```

Comment:
@rafaelraposospot no need to mock the clock. You can pass a fixed [clock](https://docs.oracle.com/javase/8/docs/api/java/time/Clock.html#fixed-java.time.Instant-java.time.ZoneId-)
```suggestion
            workflowActionAuthorizer, Clock.fixed(deploymentTime, ZoneOffset.UTC);
```